2.03 new tag element 342 plus document-link/description

amy-silcock opened the pull request:
Added Extra Documentation as per: https://github.com/IATI/IATI-Schemas/issues/342

Have added in an `<narrative>`, as a parent element of `<tag>`.

Also adds a previously missing usage example and extra documentation for `document-link/description`.

amy-silcock committed:
Add new tag element

dalepotter commented:
Changed base branch in order to force a refresh to the commit and file diff. This PR previously included changes already in the base branch.

dalepotter committed:
Fix example activity xml

There is no `description` element within a `tag` element. This removes the element to make the tests pass.

dalepotter commented on en/activity-standard/iati-activities/iati-activity/document-link/description.rst:
```diff
@@ -0,0 +1,17 @@
+Example Usage
```
`iati-activities/iati-activity/document-link/description` does not appear to relate to anything in IATI/IATI-Schemas#342 (as per the PR description).

dalepotter replied:
Appears to be related to #392 

Will amend the PR title and approve as part of this PR.

dalepotter commented on en/activity-standard/iati-activities/iati-activity/tag/description.rst:
```diff
@@ -0,0 +1,16 @@
+Example Usage
```
`iati-activities/iati-activity/tag/description` is not included in the [2.03 Upgrade - Final Technical Summary](https://docs.google.com/spreadsheets/d/1XWnzesMvLebIVLk47OIXATfhcjESVcaGnw8SJpOuCCk/edit#gid=595223309) (table lines 157-163) as implmentated in IATI/IATI-Schemas#342.

Will remove this page and references in `en/activity-standard/iati-activities/iati-activity/tag/description/narrative.rst`.

dalepotter replied:
Fixed in e7ef28b

dalepotter committed:
Remove reference to non-existant element: iati-activity/tag/description

This element was added to documentation in error.

dalepotter commented on en/activity-standard/iati-activities/iati-activity/document-link/description.rst:
```diff
@@ -0,0 +1,17 @@
+
+2.03
+^^^^
+The optional ``description`` element of a ``document-link`` in a ``result`` element was `added <https://discuss.iatistandard.org/t/document-link-description-included-2-03/841>`__.
```
Path referenced here is not consistent with the file title.

dalepotter replied:
Fixed in 471018e

dalepotter committed:
Fix path for document-link/description

Previous path refered to an in correct path (given the path that this file is located at).
